### ISSUE TITLE
Booyah! Failed

### ISSUE THREAD

**Issue by prestonmcgowan** [twigkit/tempo#92]
When I load tempjs.com the Booyah is empty.

I see this error in the JS Console:
 Failed to load resource: the server responded with a status of 410 (Gone)
http://search.twitter.com/search.json?q=%27groucho%20marx%27&rpp=2&callback=jQuery15003200380178168416_1420802403256&_=1420802403300

When loading the link above, I see the following error:
/**/jQuery15003200380178168416_1420802403256({"errors":[{"message":"The Twitter REST API v1 is no longer active. Please migrate to API v1.1. https://dev.twitter.com/docs/api/1.1/overview.","code":64}]});

**Comment by mrolafsson** [comment]
Yeah annoyingly the Twitter API now needs authentication and I haven't updated the demos. I didn't know if there was still interest in me maintaining this --  I had made good progress on the 3.0 branch which was much faster and more reliable. What do you think?

**Comment by BMeridian** [comment]
Dont maintain, build new better example showing speed improvements....

**Comment by mrolafsson** [comment]
Sure -- any thoughts on a good JSON data set?

**Comment by markddrake** [comment]
tmdb API...
